packages/nimble-components/src/card/template.ts

```diff
@@ -0,0 +1,8 @@
+import { html } from '@microsoft/fast-element';
+import type { Card } from '.';
+
+export const template = html<Card>`
+    <template>
+        <slot></slot>
+    </template>
```

Comment:
Since you aren't modifying the `<template>` element at all by adding a role, event handlers, attributes, css classes, etc, you don't need it here.

```suggestion
    <slot></slot>
```

Reply:
Done. Prettier moved it all onto one line: `export const template = html<Card>`\<slot\>\</slot\>`;`